src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/Modal.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import Portal from 'react-portal';
+import React from 'react';
+import modalStyle from './modal.scss';
+
+export default class Modal extends React.PureComponent {
+    props: {
+        isOpen: boolean,
+        children: React.Element<*>,
+        onRequestClose?: () => void,
+    };
+
+    static defaultProps = {
+        isOpen: false,
+    };
+
+    requestClose = () => {
+        if (this.props.onRequestClose) {
+            this.props.onRequestClose();
+        }
+    };
+
+    handleBackdropClick = this.requestClose;
+
+    render() {
+        return (
+            <Portal isOpened={this.props.isOpen}>
+                <div className={modalStyle.container}>
+                    <div className={modalStyle.modal}>
+                        {this.props.children}
+                    </div>
+                    <div onClick={this.handleBackdropClick} className={modalStyle.backdrop} />
```

Comment:
Wasn't the idea to move that into an own `Backdrop` component, so that it can also be used for the dropdowns?